projects/systelab-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
PrimeNg should be updated to 11 to be compatible?

Reply:
Ok, I will review it.